### PR TITLE
Package up beginning of API wrapper

### PR DESCRIPTION
**Pull request by mikearango**

### CODE REVIEW

**Comment by andrewrreed** [comment]
@mikearango Are all of those dependencies in `requirements.txt` necessary? I'm getting the following error even after adding the conda-forge channel...
image
image

**Comment by mikearango** [comment]
> @mikearango Are all of those dependencies in `requirements.txt` necessary? I'm getting the following error even after adding the conda-forge channel...
> image
> image

@areed1242 It looks like I have the `anaconda` channel as my highest priority: 
Screen Shot 2019-04-09 at 9 21 05 PM

So when I installed, it used the anaconda channel for everything. We can always just make a new env using `condo-forge` as the default. All I did was create a new environment with numpy and pandas and it installed those as dependencies. 

Screen Shot 2019-04-09 at 9 20 33 PM

**Comment by mikearango** [comment]
@areed1242 Ok. I fixed the `requirements.txt` for you

File: pyphish/api.py [hunk 1]
```diff
@@ -114,25 +116,38 @@ def get_all_shows(self):
...
-        payload = self._add_api_key_to_query_params()
-        payload['year'] = 2018
-        payload['order'] = 'DESC'
+        # initialize dataframe to hold all shows
+        all_shows = pd.DataFrame()
 
-        # request data
-        response = requests.request("GET", self.endpoint_, params=payload)
+        # query all shows by year
+        for year in range(1983, datetime.datetime.now().year+1):
```
Comment (mikearango) [inline pyphish/api.py]: We should wrap up the code in the for loop into a `get_shows_by_year()`method and then just call that in `get_all_shows()`. This looks real nice though!

Comment (andrewrreed) [reply pyphish/api.py]: good call... this is fixed with c69feca

**Comment by andrewrreed** [comment]
@mikearango I think we now have enough to pull all the data we need for v1....we should talk big picture strategy on our architecture and data workflow this week... i'll setup some time

**Pull request merged**